rest/config_legacy.go

```diff
@@ -289,7 +291,13 @@ func (config *LegacyServerConfig) validate() (errorMessages error) {
 				"unsupported.stats_log_freq_secs", 10))
 		}
 	}
-
+	if config.ServerTLSSkipVerify != nil && *config.ServerTLSSkipVerify {
+		for name, database := range config.Databases {
+			if database.CACertPath != "" {
+				errorMessages = multierror.Append(errorMessages, fmt.Errorf("cannot skip server TLS validation and use CA Cert for database %s", name))
+			}
+		}
+	}
```

Comment:
I think this validation is already handled as part of `setupServerContext` -> `config.validate` - we'll just be hitting it a bit later in the startup process.

Reply:
All done. Manually tested it and it does hit the persistent config validate function.